### ISSUE TITLE
Module paths in data/scrape/lib/codes.py

### ISSUE THREAD

**Issue by kylebgorman** [CUNY-CL/wikipron#487]
From wikipron/data/scrape, with an editable installation, I am running `./scrape.py`. The following error is raised:

`ModuleNotFoundError: No module named 'data.scrape.lib'`

I understand why this is broken (`data` will not be a package at this location) but I also don't understand why this was ever supposed to work, so I am not ready to fix it yet.

(Another small blocker for #485.)

**Comment by jacksonllee** [comment]
I wasn't able to reproduce the `ModuleNotFoundError`. After having `cd`-ed down to `data/scrape/`, I ran `./scrape.py` like you did, and I saw the scraping process got kicked off with logs like `config.py INFO: 10-Mar-23 21:24:12 - Language: 'Afar'` starting to show up in the terminal as expected. My Python environment happens to be running 3.10, and also has WikiPron installed as an editable package. What did we miss?
